src/features/houseHold/permissions/household.ts

```diff
@@ -6,7 +6,10 @@ export async function assertHouseholdWriteAccess(householdId: string) {
   const user = session?.user;
   const household = await getHousehold(householdId);
 
-  if (user && user.id == household?.ownerId) {
+  const isOwner = user && user.id === household?.ownerId;
+  const isMember = user && household?.members.find((u) => u.userId == user.id);
+
+  if (isOwner || isMember) {
     return;
   }
```

Comment:
jak nie ma usera to od razu przy sprawdzaniu sesji/usera możesz throwować żeby nie próbować pobierać potem niepotrzebnie householda